docs/quickstart.md

```diff
@@ -1,20 +1,20 @@
 # Quickstart
 
-[Here's](https://github.com/google/ksp/releases/download/1.4.20-dev-experimental-20210203/playground.zip) a sample processor that you can check out.
+[Here's](https://github.com/google/ksp/releases/download/1.4.30-1.0.0-alpha01/playground.zip) a sample processor that you can check out.
```

Comment:
Actually in your pre-release, this is artifacts.zip. not sure if that's what you want.